css/style.css

```diff
@@ -267,7 +267,7 @@ video {
   /* desktop carrousel */
   .container {
     max-width: 960px;
-    padding: 88px 0;
```

Comment:
did you make sure nothing broke when you removed this?

Reply:
yes, he spoke on Thursday with camila and jose to get it out because none of the pages was using that 88px padding